### PR TITLE
Add manifest com.github.muriloventuroso.easyssh

### PR DESCRIPTION
**Pull request by muriloventuroso**

### CODE REVIEW

File: com.github.muriloventuroso.easyssh.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "com.github.muriloventuroso.easyssh",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
```
Comment (barthalion) [inline com.github.muriloventuroso.easyssh.json]: There is also 3.28 now (fine if you aim at previous release though).

File: com.github.muriloventuroso.easyssh.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            ]
+        },
+
+        {
+            "name": "easyssh",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/muriloventuroso/easyssh.git"
```
Comment (barthalion) [inline com.github.muriloventuroso.easyssh.json]: It has to point to specific tag or commit from a stable branch.

File: com.github.muriloventuroso.easyssh.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "com.github.muriloventuroso.easyssh",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.muriloventuroso.easyssh",
+    "finish-args": [
+        "--share=ipc",
```
Comment (barthalion) [inline com.github.muriloventuroso.easyssh.json]: I assume it needs at least networking access and read-only to SSH keys. I'm not sure if it's enough to actually make ssh happy – which by the way is not bundled.

**Comment by AsciiWolf** [comment]
bot, build com.github.muriloventuroso.easyssh

**Comment by muriloventuroso** [comment]
The last few commits have fixed what was requested. Can you check again?

**Comment by AsciiWolf** [comment]
bot, build com.github.muriloventuroso.easyssh

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.muriloventuroso.easyssh

**Pull request closed without merging**